### DISCUSSION TITLE
Can't add more than 2 sharing links

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#6456] [Q&A] [answered]
**Describe the bug**
It seems each time I create a third sharing link freshRSS deletes the previously created one.

**To Reproduce**
1. Go to '/i/?c=configure&a=integration'
2. Click on 'Ajouter une méthode de partage ' '+'
3. choose something like Mastodon
4. validate
5. fill in the name and link
6. validate
7. previous method is deleted

**Environment information (please complete the following information):**
- Device:  PC
- OS:  Ubuntu 22.04
- Browser: Firefox 
- FreshRSS version:1.22.1 alwaysadata

**Comment by Alkarex** [comment]
Did you remember to click on the little + button?
![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/63802761-2bcb-4ab4-ad66-1a739404c18b)

I cannot reproduce a problem:

![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/8d4187e1-9bc9-4ab9-97ad-783395cb9405)

**Comment by ghost** [comment]
Yes indeed I tried this, no success

**Comment by Alkarex** [comment]
Could you share a screenshot, or screencast? Anything in the logs?
Could you try with our edge branch?

**Comment by ghost** [comment, marked as answer]
[solved]
I needed to add  and save all the share links at once (and not one by one).
